canonical-web-and-design/cn.ubuntu.com
# Update homepage to display spotlight article & update blogs to show featured articles

## Done

- Fixes latest-news.js to display a spotlight article
- Update blog to display 3 featured articles on the first page
- Minor design fixes and updating the style to match that of ubuntu.com/blog

## QA

- Visit 
- Go to the homepage and see that the news strip displays 3 of the latest news articles and a 4th 'spotlight' article.
- Go to /blog and see that there are 3 featured articles only on page 1
- See that the style of the blog more closely resembles ubuntu.com/blog
- Check the Chinese is correct

## Issue / Card

https://github.com/canonical-web-and-design/web-squad/issues/5300
